app/Dockerfile

```diff
@@ -7,15 +7,19 @@ ENV MM_VERSION=4.4.1
 # Build argument to set Mattermost edition
 ARG edition=entreprise
 
-# Install some needed packages
-RUN apt-get update \
-    && apt-get -y --no-install-recommends install \
-      curl \
-      jq \
-      netcat \
-      ca-certificates \
-    && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
 
+RUN apk add --no-cache \
+	bash \
+	ca-certificates \
+	curl \
+	jq \
+	libc6-compat \
+	libffi-dev \
+	linux-headers \
+	netcat-openbsd \
+	&& rm -rf /var/cache/apk/* \
```

Comment:
The `--no-cache` flag shouldn't leave anything behind here, so I'm pretty sure this line is unnecessary.

Reply:
@ccbrown is this?
`rm -rf / var / cache / apk / * \`

Reply:
I'm suggesting removing that line:

```
RUN apk add --no-cache \
	bash \
	ca-certificates \
	curl \
	jq \
	libc6-compat \
	libffi-dev \
	linux-headers \
	netcat-openbsd \
	&& rm -rf /tmp/* 
```

> As of Alpine Linux 3.3 there exists a new --no-cache option for apk. This avoids the need to use --update and remove /var/cache/apk/* when done installing packages.

– https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache